src/storage-client/src/statistics.rs

```diff
@@ -378,12 +378,13 @@ impl<T: StorageMetric + Default + Clone> SkippableGauge<T> {
         }));
 
         // If any are none, we can't aggregate.
```

Comment:
does this still make sense?

Reply:
i think so? ill make sure before I merge

Reply:
yes it makes sense! my logic was wrong before! i was creating a `Some(sum)` if any were not, instead of it none were none